### DISCUSSION TITLE
Photo item gradient background specifications ?

### DISCUSSION THREAD

**Discussion by KaKi87** [FossifyOrg/General-Discussion#413] [Q&A]
Hi,

I'm creating a web gallery component for a blog and would like to get inspired by Fossify Gallery's gradient background for photo items, as shown in the [first demo screenshot](https://github.com/FossifyOrg/Gallery/blob/master/fastlane/metadata/android/en-US/images/phoneScreenshots/1_en-US.png).

So, I looked in the code, and found the following reference :

https://github.com/FossifyOrg/Gallery/blob/f649b1ef615ab0d41433fdf04ccdda3291a6fecc/app/src/main/res/layout/photo_item_grid.xml#L64C29-L64C58

Then, I looked the declaration of this value, but didn't find it.

Help ?

Thanks

**Comment by Aga-C** [comment]
It's defined here: https://github.com/FossifyOrg/Gallery/blob/master/app/src/main/res/drawable/gradient_background.xml. The gray color is from there: https://github.com/FossifyOrg/Commons/blob/0c8d33952efbd38e818b74a9e0017954f1116564/commons/src/main/res/values/colors.xml#L10.
